### DISCUSSION TITLE
Encoded word bigger than 76 chars

### DISCUSSION THREAD

**Discussion by soywod** [TornaxO7/rfc2047-decoder#16] [General]
https://github.com/soywod/rfc2047-decoder/blob/2209238eadb0d22a543b477d09d62b880eb93c8f/src/lexer.rs#L312-L325

> An encoded word with more then 75 chars should be parsed as a normal cleartext

Is it written somewhere in the RFC or is it a rule you established yourself? It does not seem right to me, if a component is identified as an encoded word (between `=?…?=`) then it should be treated as an encoded word, not a clear text. For example, if the length of the encoded word is greater than 76 I expect the lib to throw an error instead of returning the encoded string.

**Comment by TornaxO7** [comment]
Oh, I didn't saw this discussion. I made the rule by myself, that it will be parsed as a clear text. The latest PR (https://github.com/soywod/rfc2047-decoder/pull/31) will give the dev the power to decide what should happen if an encoded word is too long and the default is, to return an error :)

**Comment by TornaxO7** [comment]
Closing due to PR https://github.com/soywod/rfc2047-decoder/pull/31
